easyInterface/Diffraction/DataClasses/DataObj/Experiment.py

```diff
@@ -324,11 +358,66 @@ def __repr__(self) -> str:
         return '{} Experimental phases'.format(len(self))
 
 
+class chi2(LoggedPathDict):
```

Comment:
We should consider to rename this class into something like RefinementType, as chi2 is associated with calculation of the goodness-of-fit.

Reply:
Modified